src/stream/stream_backend.rs

```diff
@@ -63,19 +62,19 @@ fn check_encode(
         VideoEncodeType::UNKNOWN(name) => match video_and_stream_information.video_source {
             VideoSourceType::Redirect(_) => (),
             _ => {
-        return Err(SimpleError::new(format!(
-            "Encode is not supported: {}",
-            name
+                return Err(SimpleError::new(format!(
```

Comment:
This should be done in 966e7e9dc55cf64f98ab570e19a1726891014456

Reply:
Yeah, makes sense.